server/src/monitoring/monitoring-client.js

```diff
@@ -0,0 +1,24 @@
+var winston = require('winston'),
+    fs = require('fs'),
+    APPDYNAMICS_SCRIPT = './monitoring/appdynamics-client.js';
+
+var appKey = process.env.APPDYNAMICS_RUM_KEY;
+
+if (!appKey) {
+    winston.error('APPDYNAMICS_RUM_KEY not set in environment');
+    return;
+}
+
+if (fs.existsSync(APPDYNAMICS_SCRIPT)) {
+    var adScript;
+    try {
+        adScript = String(fs.readFileSync(APPDYNAMICS_SCRIPT));
+    } catch (err) {
+        winston.error('Could not read appdynamics js file.', err.message);
+    }
+    var adScriptInjectedKey = adScript.replace('a.appKey=""', 'a.appKey="' + appKey + '"');
```

Comment:
Why continuing after error on read? You would call `replace` on `undefined`. I'd propose to just wrap everything in try/catch.